src/modules/peek/Peek.UI/MainWindow.xaml

```diff
@@ -17,11 +17,15 @@
 
     <Grid>
         <Grid.RowDefinitions>
-            <RowDefinition Height="32" />
+            <RowDefinition Height="48" />
```

Comment:
Is this breaking the edge-to-edge display? TitleBarHeight is used for window centering/resizing.

Reply:
Nope :) coz you used "ActualHeight" for calculating the max sizes, etc. I think. It is just gonna have a bit of a wider topbar: 
<img width="680" alt="image" src="https://user-images.githubusercontent.com/39350350/206325230-91638c88-6988-4648-b1fd-569c38928d94.png">